docs(VTooltip): add example for color prop (#14375)
vuetifyjs/vuetify: packages/docs/src/pages/en/components/tooltips.md

<example file="v-tooltip/prop-alignment" />

#### Color

> Tooltip contain `color` prop which allows you to select Tooltip color.

Reviewer: ```suggestion
Tooltip color can be set with the `color` prop.
```
